projects/templates/projects/issues/issue.html

```diff
@@ -0,0 +1,7 @@
+{% extends "../base.html" %}
+{% load projects_extras %}
+{% block title %}TAIPAN | Issue - {{ isd.name }}{% endblock %}
```

Comment:
`isd` should probably just be `issue`. The fox antipattern of database table name aliasing should be avoided.